### DISCUSSION TITLE
Add new CVEs 2

### DISCUSSION THREAD

**Discussion by huynh-van-tuong** [jorhelp/Ingram#104] [Q&A] [answered]
Can you add the program?
https://github.com/bigherocenter/CVE-2024-7029-EXPLOIT
https://github.com/geniuszlyy/CVE-2024-7029
https://github.com/xiaoWangSec/DaHuaPoC
https://github.com/infobyte/ezviz_lan_rce
looking forward to your help

**Comment by jorhelp** [comment, marked as answer]
Hi, do you have some cve-2024-7029 ips for me to test my code? I used the poc of these authors and did not detect any device with the cve-2024-7029 vulnerability.

**Reply by huynh-van-tuong** [reply]
it seems these pocs are not working, i run the result is not good
Can you add hikvision and dahua ipcam cves, i don't see cves 2022, 2023, 2024 in the program

**Comment by andyco2008** [comment]
> Hi, do you have some cve-2024-7029 ips for me to test my code? I used the poc of these authors and did not detect any device with the cve-2024-7029 vulnerability.

我可以给你我的fofa apikey你试试
